<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FmtFieldDlg</class>
 <widget class="QDialog" name="FmtFieldDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование поля</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Идентификатор:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Идентификатор таблицы:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Наименование:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Тип:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Размер:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Смещение:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Длина:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Знаков после запятой:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Коментарий:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="idEdit">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fmtIdEdit">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit">
         <property name="maxLength">
          <number>128</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeCombo"/>
       </item>
       <item>
        <widget class="QSpinBox" name="sizeBox">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="offsetBox">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="outlenBox">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="decpointBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="commentEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maxLength">
          <number>128</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ishiddenBox">
         <property name="text">
          <string>Скрытое поле</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="firstButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/fmt.qrc">
         <normaloff>:/img/DataContainer_MoveFirstHS.png</normaloff>:/img/DataContainer_MoveFirstHS.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prevButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/fmt.qrc">
         <normaloff>:/img/DataContainer_MovePreviousHS.png</normaloff>:/img/DataContainer_MovePreviousHS.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/fmt.qrc">
         <normaloff>:/img/DataContainer_MoveNextHS.png</normaloff>:/img/DataContainer_MoveNextHS.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lastButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/fmt.qrc">
         <normaloff>:/img/DataContainer_MoveLastHS.png</normaloff>:/img/DataContainer_MoveLastHS.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/fmt.qrc"/>
 </resources>
 <connections/>
</ui>
